pkgs/by-name/yt/yt-dlp/package.nix

```
@@ -68,25 +89,31 @@ python3Packages.buildPythonApplication rec {
       ''--prefix PATH : "${lib.makeBinPath packagesToBinPath}"''
     ];
 
-  setupPyBuildFlags = [
-    "build_lazy_extractors"
-  ];
-
   # Requires network
   doCheck = false;
 
-  postInstall = lib.optionalString withAlias ''
-    ln -s "$out/bin/yt-dlp" "$out/bin/youtube-dl"
-  '';
+  postInstall =
+    ''
+      installManPage yt-dlp.1
 
-  passthru.updateScript = [
-    update-python-libraries
-    (toString ./.)
-  ];
+      installShellCompletion \
+        --bash completions/bash/yt-dlp \
+        --fish completions/fish/yt-dlp.fish \
+        --zsh completions/zsh/_yt-dlp
 
-  meta = with lib; {
-    homepage = "https://github.com/yt-dlp/yt-dlp/";
+      install -Dm644 Changelog.md README.md -t "$out/share/doc/yt-dlp"
```

```suggestion
      install -Dm644 Changelog.md README.md -t "$out/share/doc/yt_dlp"
```

I diffed the build before and after

```diff
@@ -3390,8 +3389,9 @@
     │   └── completions
     │       └── yt-dlp
     ├── doc
-    │   └── yt_dlp
-    │       └── README.txt
+    │   └── yt-dlp
+    │       ├── Changelog.md
+    │       └── README.md
     ├── fish
     │   └── vendor_completions.d
     │       └── yt-dlp.fish
```